### PR TITLE
Max skew is calculated during runtime and not a defined fixed value.

### PR DESCRIPTION
**Pull request by Ola-Palm**
Perhaps should be done once when RX stream is set up, and stored in
struct audio.
/ Ola Palm

### CODE REVIEW

File: src/audio.c [hunk 1]
```diff
@@ -857,12 +855,14 @@ static void stream_recv_handler(const struct rtp_header *hdr,
...
 	struct audio *a = arg;
 	struct aurx *rx = &a->rx;
+	const struct config_avt *avt = &a->strm->cfg;
 	bool discard = false;
 	size_t i;
 	int wrap;
 	double wall_clock=0;
 	double dur=0;
 	double skew=0;
+  double max_skew=0;
```
Comment (alfredh) [inline src/audio.c]: the indentation here should be 1 TAB.

**Comment by alfredh** [comment]
hi @Ola-Palm 

many thanks for the patch, it looks good.

a minor ccheck error has to be fixed:

./src/audio.c:988: line is too wide (96 - max 79)

**Comment by Ola-Palm** [comment]
Still not valid?

**Comment by alfredh** [comment]
Thank you :)

**Pull request merged**